### DISCUSSION TITLE
Is there a way to find the path to the pandoc executable that quarto uses?

### DISCUSSION THREAD

**Discussion by joelostblom** [quarto-dev/quarto-cli#10540] [Q&A] [answered]
### Description

Me again. Is there a way to get quarto to report the path to the pandoc executable it is using? Something like [`pandoc_exec` for Rmarkdown](https://rmarkdown.rstudio.com/docs/reference/pandoc_exec.html).

The background is that I need to detect if RMarkdown can run from terminal on multiple different OSes and since RMarkdown relies on RStudio setting a PATH variable on startup to locate pandoc, I'm currently telling RMarkdown where pandoc is manually via the [`find_pandoc` function](https://rmarkdown.rstudio.com/docs/reference/find_pandoc.html). This function needs a path to find pandoc in, and I'm currently using a few common paths that I have seen across OSes, but it would be convenient with a command. I tried the following which seemed to work for me on Ubuntu, but a few colleagues reported it not outputing a path for them on Mac:

```bash
echo "cat(dirname(paste(rmarkdown::pandoc_exec())))" > tmp-quarto-pandoc.R
quarto run tmp-quarto-pandoc.R
```

**Comment by cscheid** [comment, marked as answer]
I don't think we have a clean way to do it, but `quarto check` could provide that. Can you create an issue? Thanks.

**Reply by joelostblom** [reply]
Thank you, issue here https://github.com/quarto-dev/quarto-cli/issues/10561
